terraform/bod_docker_acl_rules.tf

```diff
@@ -22,29 +22,7 @@ resource "aws_network_acl_rule" "docker_ingress_anywhere_via_ephemeral_ports_tcp
   to_port = 65535
 }
 
-# Allow ingress via ephemeral ports from Google DNS via UDP
-resource "aws_network_acl_rule" "docker_ingress_from_google_dns_via_ephemeral_ports_udp_1" {
-  network_acl_id = "${aws_network_acl.bod_docker_acl.id}"
-  egress = false
-  protocol = "udp"
-  rule_number = 111
-  rule_action = "allow"
-  cidr_block = "8.8.8.8/32"
-  from_port = 1024
-  to_port = 65535
-}
-resource "aws_network_acl_rule" "docker_ingress_from_google_dns_via_ephemeral_ports_udp_2" {
-  network_acl_id = "${aws_network_acl.bod_docker_acl.id}"
-  egress = false
-  protocol = "udp"
-  rule_number = 112
-  rule_action = "allow"
-  cidr_block = "8.8.4.4/32"
-  from_port = 1024
-  to_port = 65535
-}
-
-# Allow outbound HTTP, HTTPS, and FTP anywhere
+# Allow outbound HTTP, HTTPS, SMTP (587), and FTP anywhere
```

Comment:
Remove `SMTP (587)` from this comment

Reply:
You are correct, sir!  I will remove it now.

Reply:
Removed in 7ff20c3.